mill

```diff
@@ -22,8 +22,11 @@ MILL_EXEC_PATH="${MILL_DOWNLOAD_PATH}/$MILL_VERSION"
 
 if [ ! -x "$MILL_EXEC_PATH" ] ; then
   mkdir -p $MILL_DOWNLOAD_PATH
+  if [ "$MILL_VERSION" > "0.4.2" ] ; then
```

Comment:
Is this doing what you want? What happens, if we release mill `0.10.0`?

Reply:
:grimacing: 
I think `0.10.0` is still larger than `0.4.2`, but it might be smaller than `0.4.22`.
I guess we should probably take a less naive approach and split the version numbers by section and compare them separately.